src/server/hook_func.c

```diff
@@ -3148,6 +3152,9 @@ set_job_reslist(job *pjob, char *hook_name, char *msg, int msg_len,
 		log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO,
 			pjob->ji_qs.ji_jobid, msg);
 
+		snprintf(log_buffer, sizeof(log_buffer), "%s.%s=%s", ATTR_l, resc, new_rescval_str);
+		account_record(PBS_ACCT_ALTER, pjob, log_buffer);
```

Comment:
can there be multiple ATTR_l being set ? If so, wouldn't it be better to just call account_record() once for all of them?

Reply:
As I said below.  There should be one record per attribute or resource.